BUILDING.md

```diff
@@ -176,6 +176,16 @@ cmake -GXcode -T buildsystem=1 -Daudacity_use_mad="off" -Daudacity_use_id3tag=of
 
 You can use `cmake -LH` to get a list of the options available (or use CMake GUI or `ccmake`). The list will include documentation about each option. For convenience, [here is a list](CMAKE_OPTIONS.md) of the most notable options.
 
+### Building with Deep Learning 
+
+To enable deep learning features, pass `audacity_has_deep_learning=on` and `audacity_has_networking=yes` to CMake. 
+Deep Learning is only supported on Linux, macOS, and Windows 64-bit builds.
+
+### Building with ASIO support on Windows
+
+To enable ASIO support on Windows, please pass `audacity_has_asio_support=On` to CMake during the configuration. 
+ASIO is only supported on Windows and only for 64-bit builds.
+
```

Comment:
This mention of ASIO duplicates what is elsewhere in the file, and I think it's not relevant to deep learning and was mistakenly included.